packages/esm-outpatient-app/src/current-visit/current-visit.scss

```diff
@@ -0,0 +1,31 @@
+@import '~@openmrs/esm-styleguide/src/vars';
+@import '~carbon-components/src/globals/scss/vars';
+@import '~carbon-components/src/globals/scss/mixins';
+
+.visitContainer {
+  background-color: $ui-background;
+  display: grid;
+  grid-template-columns: max-content auto;
+}
+
+.bodyLong01 {
+  @include carbon--type-style('body-long-01');
+}
+
+.heading {
+  @include carbon--type-style('productive-heading-02');
+  color: var(--text-01);
```

Comment:
Does carbon provide a CSS variable for this color?. In general, we have tended to stick to the color tokens e.g `$text-01`, i would recommend we stick with that